easybuild/easyconfigs/i/ITK-SNAP/ITK-SNAP-3.6.0-foss-2017b.eb

```diff
@@ -0,0 +1,39 @@
+#
+# Author: Fenglai Liu
+# fenglai@accre.vanderbilt.edu
+# Vanderbilt University
+#
+easyblock = "PackedBinary"
+
+name = 'ITK-SNAP'
+version = '3.6.0'
+
+homepage = 'http://www.itksnap.org/pmwiki/pmwiki.php'
+description = """ITK-SNAP is a software application used to segment structures in 3D medical images, and it provides 
+semi-automatic segmentation using active contour methods, as well as manual delineation and image navigation."""
+
+# The foss toolchain is needed because the loading of binary needs the libGLU
+# therefore the toolchain is not dummy
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['https://sourceforge.net/projects/itk-snap/files/itk-snap/3.6.0']
+sources = ['itksnap-%(version)s-20170401-Linux-x86_64-qt4.tar.gz']
+checksums = ['b0818d0de4ae8295e514d02f5fccea9c144a56f088c40b2f70a6a78616954210']
+
+dependencies = [
+    ('libpng', '1.2.58', '', True),
```

Comment:
@murfreesboro You should bump this to `1.6.32` to avoid introducing conflicts...

Reply:
@boegel This installation uses a pre-compiled binary so there is no way to link statically against `libpng/1.2.x`. I vote for closing it.

Reply:
OK, closing then...